control/mpc_lateral_controller/src/mpc_lateral_controller.cpp

```diff
@@ -252,8 +252,12 @@ trajectory_follower::LateralOutput MpcLateralController::run(
     m_is_ctrl_cmd_prev_initialized = true;
   }
 
+  const bool is_driving_manually = input_data.current_operation_mode.mode == autoware_adapi_v1_msgs::msg::OperationModeState::LOCAL ||
```

Comment:
The calculation of is_driving_manually is incorrect. Even if the OperationMode is Auto, if control_mode is Manual, the steering will not be automatically controlled.

Reply:
fixed in 7e170bf70e7268c3d05db0100c40982ff047f61f:
Use the flag `is_autoware_control_enabled` ([Autoware control flag](https://autowarefoundation.github.io/autoware-documentation/pr-279/design/autoware-interfaces/ad-api/features/operation_mode/)).

Reply:
fixed in 3b04ccdfc86a60c692f8d5cd138fe2a71874596e